rm-services-rh-stub/app.py

```diff
@@ -0,0 +1,101 @@
+from flask import Flask
+from flask import Response
+from sample_stub import Sample_Stub
+from collex_stub import Collex_Stub
+from collex_instr_stub import Collex_Instr_Stub
+from case_stub import Case_Stub
+from iac_stub import Iac_Stub
+
+import logging
+app = Flask(__name__)
+
+#override the default logging configuration so that info messages get output
+
+logging.basicConfig(level=logging.INFO)
+
+
+@app.route('/')
+def hello():
+    return "Hello from the RM services stub for RH"
+
+""" Route handler to deal with getting sample attributes from the sample service. 
+The sampleid could be any sampleid that is passed in """
+
+@app.route('/samples/<sampleid>/attributes')
+def get_sample_attributes(sampleid):
+    my_sample_stub = Sample_Stub(sampleid)
+    json_data = Sample_Stub.get_sample_stub(my_sample_stub)
+    logging.info('Retrieved sample attributes for sampleid ' + str(sampleid))
```

Comment:
Can you please tell me if there is any tagging mechanism present in the project that could help in debugging a specific request or multiple requests at a time? 
This could be used in the future to analyse system's behaviour and performance